### ISSUE TITLE
Tabs collapse with android keyboard

### ISSUE THREAD

**Issue by nikolasbrandao** [aksonov/react-native-router-flux#3383]
### Version

- react-native-router-flux v4.0.6
- react v16.3.1
- react-native v0.55.4

### Expected behaviour

I would like the tabs in android to be behind the keyboard as is the behavior of the same component in ios.

### Actual behaviour

I added tabs to my app and I would like the tabs to work on both android and ios, but on android the tab collapses when the keyboard is activated.

### Code

`<Scene key="drawer" tabs tabBarPosition='bottom'
            type={ActionConst.RESET} inactiveTintColor={colors.gray_boulder} 
            activeTintColor={colors.blue_daintree} tabBarPosition={'bottom'} 
            tabBarStyle={{ backgroundColor: colors.white_solid }} >`
  `<Scene key="noticiasHome" component={Home} title="Notícias" tabBarLabel="Notícias" initial gesturesEnabled={false} icon={iconNoticias} />`
`
</Scene>`

### Image

![screen shot 2018-12-18 at 17 46 03](https://user-images.githubusercontent.com/5131154/50184059-b562c100-02f2-11e9-8704-cd0d87ff72ca.png)

**Comment by xinfang** [comment]
+1

**Comment by 1107979819** [comment]
+1

**Comment by aksonov** [comment]
Could you please check latest 4.1.0-beta.5, does it have the same issue?

**Comment by sajorahasan** [comment]
Yes @aksonov latest 4.1.0-beta.5 seems to be have the same issue.
